hypertrace-graphql-span-processing-schema/src/main/java/org/hypertrace/graphql/spanprocessing/schema/rule/filter/SpanProcessingLogicalFilter.java

```diff
@@ -15,11 +16,13 @@ public interface SpanProcessingLogicalFilter {
 
   @GraphQLField
   @GraphQLName(SPAN_PROCESSING_LOGICAL_OPERATOR_KEY)
+  @GraphQLDescription("Span processing logical operator")
```

Comment:
Can we make these descriptions more meaningful? Re-casing the field name isn't very helpful - it's the documentation equivalent of code like
```
 if (isEnabled()) { // if enabled
```

Instead we want to describe the _what_ and the _why_. How do we explain these to an end user? What values can be assigned? What are defaults, if applicable? What even is a span exclude rule? I don't see that in the documentation added here.

For example, this could say something like "The logical operator is used to combine the list of filters provided in the spanFilters field"